appscale/tools/appscale_tools.py

```diff
@@ -635,6 +635,36 @@ def reset_password(cls, options):
         "following reason: {0}".format(str(exception)))
       sys.exit(1)
 
+  @classmethod
+  def create_user(cls, options, is_admin):
+    """Create a new user with the parameters given.
+
+        Args:
+          options: A Namespace that has fields for each parameter that can be
+            passed in via the command-line interface.
+          is_admin: A flag to indicate if the user to be created is an admin user
+        Raises:
+          AppControllerException: If the AppController on the head node crashes.
+            When this occurs, the message in the exception contains the reason why
+            the AppController crashed.
+        """
+    secret = LocalState.get_secret_key(options.keyname)
+    login_host = LocalState.get_login_host(options.keyname)
+
+    username, password = LocalState.get_credentials(is_admin)
+    encrypted_password = LocalState.encrypt_password(username, password)
+
+    acc = AppControllerClient(login_host, secret)
+
+    RemoteHelper.create_user_accounts(username, encrypted_password, login_host, options.keyname)
```

Comment:
`create_user_accounts` already hashes the password, so you should just provide the plaintext password as the argument to this function.